CHANGELOG.md

```diff
@@ -27,6 +27,8 @@
         -   Refactored `CreateOperationHandler` to an interface instead of a class
         -   Renamed the services and handlers of the direct GModel library => consistent use of `GModel` prefix
         -   The `ModelState` interface no longer has an `isDirty` flag. Dirty state is now handled by the `CommandStack`
+-   [server] Default port has changed from 5007 to 0, which implies autoassignment by the OS
```

Comment:
It don`t think it's necessary to split this into bullet points. One pullet points covering both should be enough. Please also adda link to the PR at the end like this: [#42]](https://github.com/eclipse-glsp/glsp-server-node/pull/42/)